### ISSUE TITLE
Typeahead clear icon misaligned in Firefox, Safari, and IE11

### ISSUE THREAD

**Issue by ryan76** [carbon-design-system/carbon-addons-beta-react#9]
Firefox:
![clear-icon_firefox](https://user-images.githubusercontent.com/3375289/32064401-5c0e7718-ba3f-11e7-9482-c63e5807cc72.png)

Safari:
<img width="655" alt="clear-icon_safari" src="https://user-images.githubusercontent.com/3375289/32064619-f89255b4-ba3f-11e7-8fa6-c30142140b63.png">

IE11:
![clear-icon_ie11](https://user-images.githubusercontent.com/3375289/32064757-6449b090-ba40-11e7-88ab-c6c391e46f44.png)

**Comment by joshblack** [comment]
Thanks for the heads-up, will bundle up the fix for this with #11.

**Comment by ryan76** [comment]
This fixed the issue when I tested it out:

```
.bx--typeahead-clear {
  top: 50%;
  transform: translateY(-50%);
}
```
